### PR TITLE
Expose Connection::stream_priority in C API

### PR DESCRIPTION
**Pull request by kachayev**
Added `quiche_conn_stream_priority` to FFI. Covers #721.

### CODE REVIEW

**Review by LPardue — DISMISSED** [review]
Thanks! Lgtm.

File: include/quiche.h [hunk 1]
```diff
@@ -270,6 +270,10 @@ enum quiche_shutdown {
     QUICHE_SHUTDOWN_WRITE = 1,
 };
 
+// Set the priority for a stream.
```
Comment (ghedo) [inline include/quiche.h:273]:
```suggestion
// Sets the priority for a stream.
```
(to match the `Connection::stream_priority()` Rust docs)

**Review by ghedo — CHANGES_REQUESTED** [review]
Added a minor nitpick. Looks good otherwise, thanks!

**Review by ghedo — APPROVED** [review]

**Comment by ghedo** [comment]
Merged, thanks @kachayev!

**Pull request merged**